pkg/network/usm/procnet/parser_test.go

```diff
@@ -0,0 +1,175 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build linux_bpf
+
+package procnet
+
+import (
+	"net/netip"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestProcNetParsing(t *testing.T) {
+	// The example below contains the following connections:
```

Comment:
please test ipv6 as well

Reply:
Good call, I'll create a test case for that

Reply:
There was actually a bug in the IPv6 parsing 😉